docs/guides/06-spatial-aggregations.md

```diff
@@ -1,4 +1,4 @@
-## Aggregating data
+## Spatial aggregations
 
 In this guide you will explore a set of functions called **aggregations**, that allow you to extract different values from your datasets by considering either the whole set of features (*global aggregations*), just the ones displayed in the current map extent (*viewport aggregations*) or those derived from spatial aggregations (*cluster aggregations*).
```

Comment:
the title is `Spatial aggregations` but here three different types of aggs are mentioned, only one of them is called "spatial".

I think that `viewport*` agg is technically spatial, although I think that it's behavior is more similar to `global*` aggs and therefore I wouldn't separate `global*` from `viewport*` + `cluster*`

Reply:
I aggree